crates/viewer/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -420,6 +420,12 @@ Display time series data in a plot.
             *state.scalar_range.start_mut() = f64::INFINITY;
             *state.scalar_range.end_mut() = f64::NEG_INFINITY;
 
+            let entity_to_label = EntityPath::short_names_with_disambiguation(
```

Comment:
nit:

```suggestion
            let label_from_entity = EntityPath::short_names_with_disambiguation(
```

for consistency with https://github.com/rerun-io/rerun/blob/main/CODE_STYLE.md#matrices